core/src/main/java/com/linecorp/armeria/client/retry/Backoff.java

```diff
@@ -58,6 +58,53 @@ static Backoff exponential(long initialDelayMillis, long maxDelayMillis, double
     }
 
     /**
+     * Returns a {@link Backoff} that computes backoff delay which is a random value between
+     * {@code minDelayMillis} and {@code maxDelayMillis} chosen by {@link ThreadLocalRandom}.
+     */
+    static Backoff random(long minDelayMillis, long maxDelayMillis) {
+        return random(minDelayMillis, maxDelayMillis, ThreadLocalRandom::current);
+    }
+
+    /**
+     * Returns a {@link Backoff} that computes backoff delay which is a random value between
+     * {@code minDelayMillis} and {@code maxDelayMillis}.
+     */
+    static Backoff random(long minDelayMillis, long maxDelayMillis, Supplier<Random> randomSupplier) {
+        return new RandomBackoff(minDelayMillis, maxDelayMillis, randomSupplier);
+    }
+
+    /**
+     * Creates a new {@link Backoff} that computes backoff delay using one of
+     * {@link #exponential(long, long, double)}, {@link #fixed(long)} and {@link #random(long, long)} chaining
+     * with {@link #withJitter(double, double)} and {@link #withMaxAttempts(int)} from the
+     * {@code specification}. This is the format for the {@code specification}:
+     * <ul>
+     *     <li>{@code exponential=[initialDelayMillis:maxDelayMillis:multiplier]}
+     *     is for {@link Backoff#exponential(long, long, double)}
+     *     (maxDelayMillis and multiplier can be omitted)</li>
+     *     <li>{@code fixed=[delayMillis]} is for {@link Backoff#fixed(long)}</li>
+     *     <li>{@code random=[minDelayMillis:maxDelayMillis]} is for {@link Backoff#random(long, long)}</li>
+     *     <li>{@code jitter=[minJitterRate:maxJitterRate]} is for {@link Backoff#withJitter(double, double)}
+     *     (if only one jitter value is specified, it will be used for {@link Backoff#withJitter(double)}</li>
+     *     <li>{@code maxAttempts=[maxAttempts]} is for {@link Backoff#withMaxAttempts(int)}</li>
+     * </ul>
+     * The order of options does not matter, and the {@code specification} needs at least an option.
+     * If you don't specify them, the values are set by default values. This is a few examples:
+     * <ul>
+     *     <li>{@code exponential=200:10000:2.0,jitter=0.2,maxAttempts=10} (default values)</li>
+     *     <li>{@code exponential=200,jitter=0.2,maxAttempts=50} (the values for exponential is omitted)</li>
```

Comment:
I thought maxDelayMillis could be set by the default value when it's not specified. I think either is ok. Do you want me to change it?